sapi/phpdbg/tests/watch_001.phpt

```diff
@@ -2,9 +2,6 @@
 Test simple recursive watchpoint
 --SKIPIF--
 <?php
-if (PHP_INT_SIZE == 4) {
```

Comment:
Probably fixes, tests pass

Reply:
but maybe causing https://github.com/php/php-src/issues/8450 issue

Reply:
The issues may be solved, but I'd only remove that skipif clause for "master".

Reply:
if it passes for PHP-8.0, why do not fix this good behaviour for PHP-8.0 roo?

Reply:
It may pass, but would it pass in all enviroments? I mean the skip reason is "There may be flaws in the implementation of watchpoints that cause failures". This may not have been resolved generally. Maybe @krakjoe can comment on this?